### PR TITLE
Master

### PR DESCRIPTION
**Pull request by hashmis79**

### CODE REVIEW

File: task1/attitude_controller.py [hunk 1]
```diff
@@ -115,9 +114,9 @@ def drone_command_callback(self, msg):
     # Callback function for /pid_tuning_roll
     # This function gets executed each time when /tune_pid publishes /pid_tuning_roll
     def roll_set_pid(self, roll):
-        self.Kp[0] = roll.Kp * 0.06  # This is just for an example. You can change the ratio/fraction value accordingly
+        self.Kp[0] = roll.Kp * 100  # This is just for an example. You can change the ratio/fraction value accordingly
         self.Ki[0] = roll.Ki * 0.008
-        self.Kd[0] = roll.Kd * 0.3
+        self.Kd[0] = roll.Kd * 1
```
Comment (SAtacker) [inline task1/attitude_controller.py:119]: It would get beyond limits i guess

File: task1/attitude_controller.py [hunk 2]
```diff
@@ -126,9 +125,9 @@ def pitch_set_pid(self, pitch):
         self.Kd[1] = pitch.Kd * 0.3
 
     def yaw_set_pid(self, yaw):
-        self.Kp[2] = yaw.Kp * 0.06  # This is just for an example. You can change the ratio/fraction value accordingly
+        self.Kp[2] = yaw.Kp * 100  # This is just for an example. You can change the ratio/fraction value accordingly
         self.Ki[2] = yaw.Ki * 0.008
-        self.Kd[2] = yaw.Kd * 0.3
+        self.Kd[2] = yaw.Kd * 100
```
Comment (SAtacker) [inline task1/attitude_controller.py:130]: Why did you change roll and yaw scalers both ?

**Review by SAtacker — CHANGES_REQUESTED** [review]
Keep the license file

File: task1/attitude_controller.py (hunk 2 above)
Comment (hashmis79) [reply task1/attitude_controller.py:130]: Coz the speed wasnt increasing. So i thought by increasing the range of Kp would do something

**Review by SAtacker — APPROVED** [review]

File: task1/attitude_controller.py (hunk 1 above)
Comment (hashmis79) [reply task1/attitude_controller.py:119]: Error isnt going beyond 200

**Review by SAtacker — APPROVED** [review]
